Resources/NetHook2/readme.md

```diff
@@ -9,7 +9,7 @@ These messages are dumped to file, and can be analyzed further with NetHookAnaly
 #### Building
 
 1. Execute `SetupDependencies.cmd` to automatically acquire and build the zlib and protobuf headers and libraries. You will need CMake for this.
```

Comment:
Is CMake still necessary?

Reply:
Nope! (at least not directly). I've updated that section now. Good catch